core/src/main/scala/org/apache/spark/h2o/backends/internal/InternalBackendUtils.scala

```diff
@@ -86,6 +86,7 @@ object InternalBackendUtils extends InternalBackendUtils {
 
     new ArgumentBuilder()
       .add(getH2OCommonArgs(conf))
+      .add(getLoginArgs(conf))
```

Comment:
I know it's WIP -> just suggestion -> We could move `.add(getLoginArgs(conf))` to the `getH2OCommonArgs(conf)` as those are options shared with worker nodes & client and remove it from ` getH2OWorkerArgs` and `getH2OWorkerAsClientArgs`